mlem/contrib/catboost.py

```diff
@@ -25,8 +25,11 @@ class CatBoostModelIO(ModelIO):
 
     type: ClassVar[str] = "catboost_io"
     classifier_file_name: ClassVar = "clf.cb"
+    """filename for catboost classifier"""
     regressor_file_name: ClassVar = "rgr.cb"
+    """filename for catboost classifier"""
     model_type: CBType = CBType.regressor
+    """type of catboost model"""
```

Comment:
Yes, they are visible in some commands help output.
I also plan to add schema docs to mlem.ai

Reply:
How can we tell which docstrings will be visible to users in `--help` output?

Reply:
If you can `declare` the class or class is used as a type for any level sub-field of a class that you can declare. Also, this will be visible as `mlem types ...` output, so lets say all of them